apps/api/services/zpe/enroll.py

```
@@ -0,0 +1,106 @@
+        self.redis = redis or get_redis_connection()
+
+    def create_token(
+        self,
+        *,
+        ttl_seconds: Optional[int] = None,
+        label: Optional[str] = None,
+    ) -> EnrollToken:
+        settings = get_zpe_settings()
+        ttl = ttl_seconds if ttl_seconds is not None else settings.enroll_token_ttl_seconds
+        if ttl <= 0:
+            raise ValueError("ttl_seconds must be >= 1")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)